doc/adrs/211_unified_project_creation.adoc

```diff
@@ -0,0 +1,22 @@
+= ADR-211 - Unified project creation
+
+== Context
+
+We want to unify the project creation.
+In the frontend, whenever the user want to create a project, no matter it is a blank project or a project initialized from a template, it goes to the same view.
+
+== Decision
+
+We will only keep the `CreateProjectInput` and add the `templateId` and `libraries`.
+
+We will no longer need for the `IProjectTemplateApplicationService` which is used to create the project then initialize the project content.
+The project content will be initialized thanks to a `@TransactionalEventListener` listening for the event emit when a project is created.
+
+== Status
+
+Proposed
+
+== Consequences
+
+- Downstream project that have provided their own implementation of `IProjectTemplateApplicationService` will need to listen to `ProjectCreatedEvent` instead.
+- Because the project content is initialized in reaction of a project creation, we will not be able to open a representation directly after the project creation from a template.
```

Comment:
You can mention here that it will be possible thanks to the PR to configure the workbench views (and representation) to open when the workbench is opened